janusgraph-dist/pom.xml

```diff
@@ -633,6 +520,7 @@
                             <configuration>
                                 <forkCount>1</forkCount>
                                 <reuseForks>false</reuseForks>
+                                <argLine>-Xmx1024m</argLine>
```

Comment:
Why did you add this line and why didn't we need it previously?

Reply:
I will check